Program.cs

```diff
@@ -1,10 +1,29 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿//An abecedarian word is a word where all its letters are arranged in alphabetical order. 
+//Given an array of words, create a function which returns the longest abecedarian word. 
+//If no word in an array matches the criteria, return an empty string. 
+
+Console.WriteLine ("Enter no. words to check the longest abecedarian word: ");
+bool input = int.TryParse (Console.ReadLine (), out int count);
+List<string> words = new List<string> ();
+string sorted = "";
+List<string> abecedarian = new List<string> ();
+if (input == true) {

```

Comment:
Initialize words and abecedarian list inside if statement. No need to initialize on both the sides.